src/components/materialization/SourceCapture/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Stack, Typography } from '@mui/material';
+import ExternalLink from 'components/shared/ExternalLink';
+import { FormattedMessage } from 'react-intl';
+import SelectCapture from './SelectCapture';
+import SourceCaptureChip from './SourceCaptureChip';
+
+const DOCS_LINK =
+    'https://docs.estuary.dev/concepts/materialization/#using-sourcecapture-to-synchronize-capture-and-materialization-bindings';
+
+function SourceCapture() {
+    return (
+        <Stack spacing={1} sx={{ mt: 2, mb: 3 }}>
```

Comment:
This subsection _feels_ a little too close to the binding selector. A bottom margin of 32 or 40 pixels gives the two subsections ample breathing room.

**32-pixel Bottom Margin**

<img width="897" alt="source_capture-bottom_margin_32" src="https://github.com/estuary/ui/assets/77648584/d5d935fe-9216-45e2-823a-9f5dcef63b3d">

<br />
<br />

**40-pixel Bottom Margin**

<img width="895" alt="source_capture-bottom_margin_40" src="https://github.com/estuary/ui/assets/77648584/ebc96538-5ad2-412b-8289-72a79ba01893">